src/lein_monolith/task/graph.clj

```diff
@@ -19,15 +20,16 @@
 
 (defn graph
   "Generate a graph of subprojects and their interdependencies."
-  [project]
+  [project opts]
   (require 'rhizome.viz)
   (let [visualize! (ns-resolve 'rhizome.viz 'save-graph)
         [monolith subprojects] (u/load-monolith! project)
+        targets (target/select monolith subprojects opts)
         dependencies (dep/dependency-map subprojects)
         graph-file (io/file (:target-path monolith) image-name)]
     (.mkdir (.getParentFile graph-file))
     (visualize!
-      (keys dependencies)
+      (or (seq targets) (keys dependencies))
```

Comment:
I'd rather abort with an error if zero projects are selected, rather than falling back to all projects. The following scenario, if stumbled upon, would seem counterintuitive:
```
$ lein monolith graph :in foo
# (graphs just foo)

$ lein monolith graph :in foo :skip foo
# (graphs all projects)
```

Reply:
Right, I forgot that the targeting code already handled returning all of the project keys if nothing was specified.